python/setup.py

```diff
@@ -2,28 +2,30 @@
 from version import get_git_version
 
 setup_args = dict(
-  name = 'swiftnav',
-  version = get_git_version(),
-  description = 'Python bindings to the libswiftnav library.',
-  license = 'LGPLv3',
-  url = 'http://www.swiftnav.com',
-  author = 'Swift Navigation Inc.',
-  author_email = 'dev@swiftnav.com',
-  maintainer = 'Swift Navigation',
-  maintainer_email = 'dev@swiftnav.com',
-  packages = ['swiftnav'],
+    name='swiftnav',
```

Comment:
The formatting has changed to to auto-formatting (autopep.8 --indent-size 2). Shall it be 2 here and below? Do we have any other non-default settings (other than indentation size)?